zone/client_packet.cpp

```diff
@@ -14278,23 +14275,6 @@ void Client::Handle_OP_ShopPlayerBuy(const EQApplicationPacket *app)
 		parse->EventPlayer(EVENT_MERCHANT_BUY, this, export_string, 0);
 	}
 
-	if (player_event_logs.IsEventEnabled(PlayerEvent::MERCHANT_PURCHASE)) {
```

Comment:
Was this duplicate ?

Reply:
It appeared to be yes.  You can see the other version around line 14193.